docs/workflows.md

```diff
@@ -15,6 +19,25 @@ Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remot
 
 To work around CORS errors, you need to upload your data to a server that does not impose restrictions on resource sharing. We recommend using [GitHub Gists](https://gist.github.com/), which allows users to upload files that can be downloaded within Iodide. After uploading your data into a gist, you can access the data by copying the URL for the "raw" version of your gist, which can be accessed at `https://gist.githubusercontent.com/{your user name}/{your gist id}/raw`
```

Comment:
s/We recommend using [GitHub.../We recommend using either the Iodide server for storing data (see below) or [GitHub.../